rpc/schemas/common/data/AngularVelocity.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "properties": {
+    "$type": { "const": "dk.cachet.carp.angularvelocity" },
```

Comment:
I think we decided on `__type` - should be consistently used here as well

Reply:
Yup. Thought of that while adding `$type` everywhere. Unfortunately JSON Schema doesn't have variables, so I'll have to change it everywhere after.

Just didn't belong in this PR. Unit tests should catch any omissions when I change this.